### ISSUE TITLE
Firebase-tools v13.11.0 has crashlytics buildtools v3 that does not work on Intel-based Macs

### ISSUE THREAD

**Issue by Altair4Ru** [firebase/firebase-tools#7292]
### [REQUIRED] Environment info
**firebase-tools:** 13.11.0+
**Platform:** MacOS

### [REQUIRED] Test case
The thing is, Crashlytics buildtools v3 now have dump_syms.bin for MacOS built for arm64 arch only, while it should be built as a universal binary
```
.//firebase-crashlytics-buildtools-2.9.9/dump_syms/macos/dump_syms.bin:   Mach-O 64-bit executable x86_64
.//firebase-crashlytics-buildtools-3.0.0/dump_syms/macos/dump_syms.bin:   Mach-O 64-bit executable arm64
```

### [REQUIRED] Steps to reproduce
Just call firebase-tools -- firebase crashlytics:symbols:upload --app={app_id_here} on Intel-based Mac

### [REQUIRED] Expected behavior

The symbols got uploaded without any errors

### [REQUIRED] Actual behavior

```
[CRASHLYTICS LOG DEBUG] Crashlytics Buildtools initialized.
[CRASHLYTICS LOG DEBUG] Extracting dump_syms from dump_syms/macos/dump_syms.bin to /<ProjectPath>/.crashlytics/dump_syms.bin
[CRASHLYTICS LOG DEBUG] Breakpad symbol generator initialized: /<ProjectPath>/.crashlytics/dump_syms.bin
...
java.io.IOException: Cannot run program "/<ProjectPath>/.crashlytics/dump_syms.bin": error=86, Bad CPU type in executable
  at java.lang.ProcessBuilder.start(ProcessBuilder.java:1048)
  at com.google.firebase.crashlytics.buildtools.ndk.internal.breakpad.BreakpadSymbolGenerator.generateSymbols(BreakpadSymbolGenerator.java:119)
  at com.google.firebase.crashlytics.buildtools.Buildtools.generateNativeSymbolFiles(Buildtools.java:306)
  at com.google.firebase.crashlytics.buildtools.Buildtools.generateNativeSymbolFiles(Buildtools.java:280)
  at com.google.firebase.crashlytics.buildtools.CommandLineHelper.executeGenerateSymbols(CommandLineHelper.java:193)
  at com.google.firebase.crashlytics.buildtools.CommandLineHelper.executeCommand(CommandLineHelper.java:125)
  at com.google.firebase.crashlytics.buildtools.CommandLineHelper.main(CommandLineHelper.java:72)
  at com.google.firebase.crashlytics.buildtools.Buildtools.main(Buildtools.java:114)
Caused by: java.io.IOException: error=86, Bad CPU type in executable
  at java.lang.UNIXProcess.forkAndExec(Native Method)
  at java.lang.UNIXProcess.<init>(UNIXProcess.java:247)
  at java.lang.ProcessImpl.start(ProcessImpl.java:134)
  at java.lang.ProcessBuilder.start(ProcessBuilder.java:1029)
  ... 7 more
```

**Comment by aalej** [comment]
Hey @Altair4Ru, thanks for reporting this. Some of the folks from our team with an macOS Intel machine were able to reproduce the issue. Let me raise this to our engineering team so they can take a look.

**Comment by mrober** [comment]
I have a fix for this, it will be included in the next version of the Crashlytics buildtools which should release in the next couple weeks. For now, I prepared a prerelease with the universal macOS binary, you can manually tell the Firebase CLI tool to use by:

1. Download [firebase-crashlytics-buildtools-3.0.2-prerelease01.jar.zip](https://github.com/user-attachments/files/15743785/firebase-crashlytics-buildtools-3.0.2-prerelease01.jar.zip) and extract it.

2. Tell the Firebase CLI tool to use the jar by setting the environment variable:
    
    ```bash
    export CRASHLYTICS_LOCAL_JAR=/path/to/firebase-crashlytics-buildtools-3.0.2-prerelease01.jar
    ```

3. Run the Firebase CLI tool in debug mode:
    
    ```bash
    firebase crashlytics:symbols:upload --debug --app=FIREBASE_APP_ID /path/to/symbols
    ```
    
    If you see a line in debug like *"[CRASHLYTICS LOG DEBUG] Skipping dumpsyms extraction, file exists: /path/to/.crashlytics/dump_syms.bin"*, delete that file. It is the old dump_syms binary from the old buildtools jar. Then try this step again.

**Comment by joehan** [comment]
Fixed by #7357
